### ISSUE TITLE
comp.pt issues with year_trt variable

### ISSUE THREAD

**Issue by siddharth-bharath** [nutnetadmin/nutnet-data-tracker#109]
in comb-by-plot-09-April-2018.csv
the site comp.pt has data from year 2012-2016 that all have their value of year_trt = 0

**Comment by nutnetadmin** [comment]
whoa this is really strange

**Comment by nutnetadmin** [comment]
example

select * from comb_by_plot where site_code = 'comp.pt' and year = 2013 and plot = 1;

yields two exactly identical rows. 

is this a problem in the comb_by_plot view?

**Comment by nutnetadmin** [comment]
not a problem with the view.
I suspect that this issue cropped up when I uploaded comp.pt soil data, which included data from plots 37-39, "extra" plots outside a larger herbivore exclosure fence. I must have appended on plot IDs for those plots in the database without copying over the site-level information (first_nutrient_year). I updated the plot table to include this information. Now the site_year view and all other views (including comb_by_plot & friends) should be free of these duplicates.

see:
fix-comp-plots-issue-109.sql
for a record of the fix

thanks siddharth!
